src/lazyload-image.ts

```diff
@@ -5,20 +5,19 @@ import 'rxjs/add/operator/mergeMap';
 import 'rxjs/add/operator/catch';
 import 'rxjs/add/observable/of';
 import { Observable } from 'rxjs/Observable';
-import { getScrollListener } from './scroll-listener';
 
-function isVisible(element: HTMLElement, threshold = 0, _window = window) {
+function isVisible(element: HTMLElement, threshold = 0, window) {
```

Comment:
Please choose an other name for `window`. It is quite confusing we we are using a variable `window` but is is not the global window variable. Furthermore, we should add a type to the variable, like:
```js
function isVisible(element: HTMLElement, threshold = 0, _window: Window) {
```